masterlist.yaml

```diff
@@ -2406,15 +2406,15 @@ plugins:
       - type: say
         content:
           - lang: en
-            text: 'For safe mod uninstall see special uninstallation instructions on [mod page](http://www.nexusmods.com/skyrim/mods/12933).'
+            text: 'For safe mod uninstall see special uninstallation instructions on [mod page](https://www.nexusmods.com/skyrim/mods/12933).'
           - lang: de
-            text: 'Für eine sichere Deinstallation der Mod lesen Sie bitte die Deinstallationsanleitung auf der [Mod Seite](http://www.nexusmods.com/skyrim/mods/12933).'
+            text: 'Für eine sichere Deinstallation der Mod lesen Sie bitte die Deinstallationsanleitung auf der [Mod Seite](https://www.nexusmods.com/skyrim/mods/12933).'
           - lang: es
-            text: 'Para desisntalar de manera segura por favor ve la [página del mod](http://www.nexusmods.com/skyrim/mods/12933).'
+            text: 'Para desisntalar de manera segura por favor ve la [página del mod](https://www.nexusmods.com/skyrim/mods/12933).'
           - lang: ja
-            text: '安全にアンインストールするには[Modのページ](http://www.nexusmods.com/skyrim/mods/12933)に記載されている専用のアンインストール手順を確認してください。'
+            text: '安全にアンインストールするには[Modのページ](https://www.nexusmods.com/skyrim/mods/12933)に記載されている専用のアンインストール手順を確認してください。'
           - lang: ru
-            text: 'Для безопасного удаления мода смотрите инструкции по удалению на [странице мода](http://www.nexusmods.com/skyrim/mods/12933) или на [странице локализации](http://modgames.net/load/247-1-0-10650).'
+            text: 'Для безопасного удаления мода смотрите инструкции по удалению на [странице мода](https://www.nexusmods.com/skyrim/mods/12933) или на [странице локализации](https://modgames.net/load/247-1-0-10650).'
```

Comment:
The modgames.net link appears to be dead. That part can probably just be removed from the translation.

Reply:
Good catch, will do so.